### PR TITLE
SUPESC-769 Install the Product Rating and Reviews feature

### PR DESCRIPTION
**Pull request by artem-png**
Ticket: https://spryker.atlassian.net/browse/SUPESC-769

### CODE REVIEW

**Review by kkicheglovspryker — APPROVED** [review]

**Review by mikost7 — DISMISSED** [review]
ok

File: _includes/pbc/all/install-features/202307.0/install-the-product-rating-and-reviews-feature.md [hunk 1]
```diff
@@ -1,201 +1,1212 @@
 
 
-## Prerequisites
+This document describes how to integrate the [Product Rating and Reviews](/docs/pbc/all/ratings-reviews/{{page.version}}/ratings-and-reviews.html) feature into a Spryker project.
 
-To prepare your project to work with Product Reviews:
```
Comment (VadymSachenko) [inline _includes/pbc/all/install-features/202307.0/install-the-product-rating-and-reviews-feature.md:5]: So from now on, we don't need the prerequisites section for some reason?

File: _includes/pbc/all/install-features/202307.0/install-the-product-rating-and-reviews-feature.md [hunk 2]
```diff
@@ -1,201 +1,1212 @@
...
+     * @return string|null
+     */
+    public function getEventQueueName(): ?string
+    {
+        return PublisherConfig::PUBLISH_QUEUE;
     }
+}
 ```
 
+5. Set up resync feature:
```
Comment (VadymSachenko) [inline _includes/pbc/all/install-features/202307.0/install-the-product-rating-and-reviews-feature.md:278]: do you mean this?
```suggestion
5. Set up and resync the feature:
```

File: _includes/pbc/all/install-features/202307.0/install-the-product-rating-and-reviews-feature.md [hunk 3]
```diff
@@ -1,201 +1,1212 @@
...
-        }
+        return [
+            new ProductReviewEventResourceQueryContainerPlugin(),
+        ];
     }
+}
 ```
 
-### Displaying an Average Rating
+7. Register the event subscriber:
```
Comment (VadymSachenko) [inline _includes/pbc/all/install-features/202307.0/install-the-product-rating-and-reviews-feature.md:338]: Same - is there an official confirmation that his name is spelled correctly?

File: _includes/pbc/all/install-features/202307.0/install-the-product-rating-and-reviews-feature.md (hunk 1 above)
Comment (artem-png) [reply _includes/pbc/all/install-features/202307.0/install-the-product-rating-and-reviews-feature.md:5]: This is an old way. Now we have another Prerequisites section

File: _includes/pbc/all/install-features/202307.0/install-the-product-rating-and-reviews-feature.md (hunk 2 above)
Comment (artem-png) [reply _includes/pbc/all/install-features/202307.0/install-the-product-rating-and-reviews-feature.md:278]: You are right. Fixed to 'Add synchronization plugins:' (we have this wording everywhere)

File: _includes/pbc/all/install-features/202307.0/install-the-product-rating-and-reviews-feature.md [hunk 4]
```diff
@@ -1,201 +1,1212 @@
...
+    protected function getSynchronizationDataPlugins(): array
+    {
+        return [
+            new ProductReviewSynchronizationDataPlugin(),
+        ];
+    }
+}
+```
+
+6. Set up event trigger feature:
```
[the remark this answers is not in the document]
Comment (artem-png) [reply _includes/pbc/all/install-features/202307.0/install-the-product-rating-and-reviews-feature.md:308]: Changed to 'Enable event trigger plugins to be able to re-trigger publish events.', as we have these wording almost everywhere. Feature link - https://docs.spryker.com/docs/scos/dev/back-end-development/data-manipulation/data-publishing/implement-event-trigger-publisher-plugins.html

File: _includes/pbc/all/install-features/202307.0/install-the-product-rating-and-reviews-feature.md (hunk 3 above)
Comment (artem-png) [reply _includes/pbc/all/install-features/202307.0/install-the-product-rating-and-reviews-feature.md:338]: This is an old way to listen to the events. But we don't have a new way implemented. Reference to the old way:
https://docs.spryker.com/docs/scos/dev/back-end-development/data-manipulation/event/listen-to-events.html#listen-to-events-with-direct-listeners

**Review by VadymSachenko — APPROVED** [review]

**Pull request merged**